src/com.company/backtracking/PalindromePartitioning.java

```diff
@@ -0,0 +1,38 @@
+package backtracking;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PalindromePartitioning {
+
+    void palindromeParitioning(String string, int index, List<String> currentList, List<List<String>> result) {
+        if(index >= string.length()) {
+            result.add(new ArrayList<>(currentList));
+            return;
+        }
+
+        for(int i = index; i< string.length(); i++) {
```

Comment:
Provide proper alignment. Use command + shift + enter.

Reply:
formatted the code